common/src/main/java/com/revenuecat/purchases/common/APIKeyValidator.kt

```diff
@@ -0,0 +1,64 @@
+package com.revenuecat.purchases.common
+
+import androidx.annotation.VisibleForTesting
+import com.revenuecat.purchases.Store
+import com.revenuecat.purchases.strings.ConfigureStrings
+
+private const val GOOGLE_API_KEY_PREFIX = "goog_"
+private const val AMAZON_API_KEY_PREFIX = "amzn_"
+
+class APIKeyValidator {
+
+    private enum class APIKeyPlatform {
+        GOOGLE,
+        AMAZON,
+        LEGACY,
+        OTHER_PLATFORM
+    }
+
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    enum class ValidationResult {
```

Comment:
if we move this class to the `purchases` module, we can make them internal right? maybe it's worth to do it just for that reason.